### ISSUE TITLE
Make activity directive start time relative to another activity directive start time

### ISSUE THREAD

**Issue by camargo** [NASA-AMMOS/aerie#449]
We need to be able to specify the start time of an activity directive relative to the start time of another activity directive plus or minus some offset.

**Comment by mattdailis** [comment]
Related (though a bit out of scope for this ticket): https://jira.jpl.nasa.gov/browse/AERIE-1853
> As of writing, a CoexistenceGoal with a TimeExpression of startsAtEnd will place an activity that starts at the end of the identified window.
>
> Imagine that the identified window is defined by the span of an activity instance:
>
> For each A place a B such that B starts at the end of A.
>
> In the simulation engine, this means that the last instant of A will be executed in parallel with the first instant of B. This is probably not what a user would ever want - we think the expected behavior is that A terminates before B starts.
>
> There is no way to represent this in our plan - one option is to expand the representation of plans to allow specifying that an activity starts X duration after the end of another activity. (Note: this means the start offset of an activity from the start of the plan becomes unknown until after simulation)

**Comment by Twisol** [comment]
> We need to be able to specify the start time of an activity directive relative to the start time of another activity directive plus or minus some offset.

Ooh, ooh, I have opinions about this one! 👀 I discussed this with somebody (Matt?) some months back, at least at the level of input to the simulation engine. It already supports this from *within* the simulation; we just don't expose it for input directives.
